posts/views.py

```diff
@@ -32,3 +34,23 @@ def list(self, request) -> Response:
 
         serializer = PostSerializer(queryset, many=True)
         return Response(serializer.data)
+
+
+class LikeDislikeViewSet(viewsets.ModelViewSet):
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, pk: int):
+        response = "Setted"
+        is_like = not ("dislike" in str(request))
```

Comment:
probably smth like 
```suggestion
        is_like = not "dislike" in request.path
```

Reply:
But the idea to have 2 urls but 1 common view is great!

Reply:
Done